### DISCUSSION TITLE
Filter with and?

### DISCUSSION THREAD

**Discussion by Pulsdesign** [Budibase/budibase#6934] [Help] [answered]
Hi, 

I have a table with some data in it and I want to show a new table only showing some of this based on a condition. I have a column with Version: 1,2,3,4,5,6 etc. connected to some data in a row.. 

Now I want to create a new table with a filter that only shows Version 1,2,3 data and not the data 4,5,6 - I have tried to put all conditions in a filter but it only shows the laster version (3).

The Versions are standing in a dropdown-field (no multi-dropdown)

Can anyone help me out with this?

thx a lot

**Comment by melohagan** [comment]
Hi @Pulsdesign 

Could you provide some screenshots of what you've tried please? 
It would be useful to see your tables and visually what you're trying to accomplish.

**Reply by Pulsdesign** [reply]
Hi Thx for reply...

here is the screenshot

<img width="1229" alt="Bildschirmfoto 2022-07-28 um 12 07 02" src="https://user-images.githubusercontent.com/7311029/181480876-7ae1c390-91ad-462a-ac86-6abc066ae1d2.png">

I want to show the range like into the filter list - but it only shows the last one (red).

thx for help

**Reply by melohagan** [reply]
Thanks @Pulsdesign There is an open ticket for this: https://github.com/Budibase/budibase/issues/2585

Also recently I've done some work on the CONTAINS filter for arrays which might be useful in this scenario: https://github.com/Budibase/budibase/pull/6931

It still has to go through QA though, so it may not be available until the release after the one coming up this month.

**Reply by Pulsdesign** [reply]
Hi, thx for the topics but I don't have a multi-select field for that because it can only be one value... so the CONTAINS doesn't show up at my selection.

Is there another way to sort that out?

<img width="1036" alt="Bildschirmfoto 2022-07-28 um 14 26 04" src="https://user-images.githubusercontent.com/7311029/181504753-a27df982-f041-46cb-bd12-bc8ce342801e.png">

**Reply by melohagan** [reply]
To clarify, which data type is 'WordPress Version' in your Budibase table?

**Reply by Pulsdesign** [reply]
> To clarify, which data type is 'WordPress Version' in your Budibase table?

Hi - it's an options datatype :) thx for helping us out!

**Comment by melohagan** [comment, marked as answer]
Thanks @Pulsdesign 

At the moment there is an 'Is in' filter for string and number types. I'll need to add it for the options type as well.
I've opened an issue here: https://github.com/Budibase/budibase/issues/6975
